typing/typedecl_variance.ml

```diff
@@ -94,7 +94,7 @@ let compute_variance env visited vari ty =
     | Tfield (_, _, ty1, ty2) ->
         compute_same ty1;
         compute_same ty2
-    | Tsubst ty ->
+    | Tsubst (ty, _) ->
```

Comment:
Can we assert `= None` here?

Reply:
I think asserting false on the whole `Tsubst` case would be clearer.

Reply:
Done